src/main/java/com/kryptnostic/datastore/cassandra/CommonColumns.java

```diff
@@ -85,7 +85,7 @@
     SECURABLE_OBJECTID( DataType.uuid() ),
     SRC( DataType.set( DataType.uuid() ) ),
     STATUS( DataType.text() ),
-    TIME_ID( DataType.uuid() ),
+    TIME_ID( DataType.timeuuid() ),
```

Comment:
I propose to create a new `TIME_UUID( DataType.timeuuid() )` instead, because `TIME_ID` is currently used in logging, and it isn't necessarily a time uuid. This change may break `AuditLogging`